src/v/config/configuration.cc

```diff
@@ -1527,6 +1527,46 @@ configuration::configuration()
       "value of `topic_partitions_per_shard` multiplied by 2 is used.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::nullopt)
+  , cloud_storage_cache_chunk_size(
+      *this,
+      "cloud_storage_cache_chunk_size",
+      "Size of chunks of segments downloaded into cloud storage cache. Reduces "
+      "space usage by only downloading the necessary chunk from a segment.",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
+      16_MiB)
+  , cloud_storage_hydrated_chunks_per_segment_ratio(
```

Comment:
This property could be problematic.  It makes the total number of hydrated chunks dependent on the segment size: a topic with 1024MB segments will be permitted 8x more chunks hydrated than a topic with 128MB segments.  That makes me worry about testing etc: we might test a system with default-sized segments, and then have issues later when someone uses larger segments.

To robustly manage resource limits (things like number of file handles, any other per-FD overhead), we would ideally have a shard-wide limit on the absolute number of hydrated chunks.  It might make sense to do a followup to this PR that revises how the chunk eviction works.

Reply:
I have set the default to eager eviction for now.